src/ResultWriter/inioutput_seissol.f90

```diff
@@ -153,6 +153,16 @@ SUBROUTINE inioutput_SeisSol(time,timestep,pvar,cvar,EQN,IC,MESH,MPI,      &
     do i = EQN%nVar+1, EQN%nVarTotal
       outputMaskInt(i) = 0
     end do
+
+    !plastic strain
+    do i = 1, 7
+        if (io%PlasticityMask(i)) then
+            outputMaskInt(EQN%nVarTotal + i) = 1
+        else
+            outputMaskInt(EQN%nVarTotal + i) = 0
+        end if
+    end do

```

Comment:
Why don't you pass on plasticityMask to c_interoperability_initializeIO? Would be much nicer

Reply:
yes you are right

Reply:
I tried that, but now get:
```
/hppfs/work/pr45fi/di73yeq4/SeisSol/src/ResultWriter/inioutput_seissol.f90(171): error #6633: The type of the actual argument differs from the type of the dummy argument.   [PLASTICITYMASK]
        i_plasticityMask=io%PlasticityMask,     &
----------------------------^
```
(see last commit)